### DISCUSSION TITLE
WIX installation for mcr.microsoft.com/dotnet/framework/sdk docker Images

### DISCUSSION THREAD

**Discussion by AndriyKarpyak** [wixtoolset/issues#6941] [Questions]
Hello and thank You all for a great job!

We have already been using the WIX to create the installers for our project for 10 years. To build the project we are using the latest WIX installer with WiX Toolset Visual Studio Extension without any problems.

Recently, it was decided to update our CI/CD and use the docker containers to build the project. At the moment we are using [.NET Framework SDK](https://hub.docker.com/_/microsoft-dotnet-framework-sdk/) image as a base to create our build container. That image is preinstalled with MSBuild but without Visual Studio.

To install WiX in docker we are using an archive with binaries and the next script:

```
ADD wix311-binaries.zip /3party/wix311-binaries.zip
RUN Expand-Archive C:\3party\wix311-binaries.zip C:\Build_Tools\WiX\wix311
ENV WixToolPath "C:/Build_Tools/WiX/wix311"
ENV WixSdkPath "C:/Build_Tools/WiX/wix311/sdk"
```

Also, we are forced to update *.wixproj files to update references to the WiX targets. But problems still are present.

**What is the correct way to install and configure WiX for MSBuild for cases when the MSBuild is installed without Visual Studio as in the case with the mcr.microsoft.com/dotnet/framework/sdk image?**

**Comment by robmen** [comment]
I think if you make sure your .wixprojs are using the very latest templates then everything should just work without VS.
